### PR TITLE
Add devices pricing page

### LINKED ISSUE #5865: [New page] devices - "Ubuntu IoT and Devices - plans and pricing"
This is a new page request from Galem. It outlines the prices for the IOT and devices. There are some unresolved comments, but we have been told; _"Most of the open comments are requests for clarifications, or require information that we don't have yet. It is Mark who requested me to publish it."_

- [Copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit?usp=drivesdk)
- [Trello Card](https://trello.com/c/nMOPV0GZ/419-devices-plan-and-pricing)

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added a new page at /pricing/devices, based on this [copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit#heading=h.l83dv5879ajz)
- Added a card for the page to /pricing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Visit: http://0.0.0.0:8001/pricing/devices
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit#heading=h.l83dv5879ajz), and is laid out correctly.

## Issue / Card

Fixes #5865

### CODE REVIEW

**Comment by sowasred2012** [comment]
@anasereijo's previous design feedback from https://github.com/canonical-web-and-design/ubuntu.com/pull/5874:

@sowasred2012 there are a few things to fix and many of the headings weren't matching the copydoc. See it below:

- [x] part I



- [x] part II



- [x] part III



- [x] part IV



- [x] part V



- [x] part VI



- [x] part VII

**Comment by pmahnke** [comment]
@sowasred2012 and @anasereijo two comments:

1. I think the certified logo looks too big
2. All the numbers in the 'Security updates and app store services' table should align right

also, we need to create a box on /pricing or else this page will be hidden.

File: templates/pricing/devices.html [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+{% block title %}Ubuntu IoT and Devices | plans and pricing{% endblock %}
+{% block meta_description %}Canonical enables sophisticated devices to be designed, delivered and operated at scale with Ubuntu and Ubuntu Core, the all-snap solution to IoT software delivery and security.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h1>Ubuntu IoT and device services</h1>
+    <p>Ubuntu is the new standard for embedded Linux development and the intelligent edge. Get ten years of security coverage and dedicated app stores for your smart connected devices.</p>
+    <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
```
Comment (pmahnke) [inline templates/pricing/devices.html]: clicking this didn't open the modal

File: templates/pricing/devices.html [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+            <th class="u-align--right">1 week</th>
+            <th class="u-align--right">48 hours</th>
+            <th class="u-align--right">24 hours</th>
+            <th class="u-align--right">6 hours</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr>
+            <td>1,000</td>
```
Comment (pmahnke) [inline templates/pricing/devices.html]: I think these all need `u-align--right` as well?

File: templates/pricing/devices.html [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+        </thead>
+        <tbody>
+          <tr>
+            <td>Physical server</td>
+            <td class="u-align--center">$750</td>
+            <td class="u-align--center">$1,500</td>
+          </tr>
+          <tr style="background-color: #e5e5e5">
+            <td>Phone and ticket support</td>
+            <td class="u-align--center">Monday - Friday</td>
```
Comment (pmahnke) [inline templates/pricing/devices.html]: on https://ubuntu.com/legal/ubuntu-advantage-service-description we changed Monday-Friday to 24/5 we should do the same here and update the copy doc

File: templates/pricing/devices.html [hunk 4]
```diff
@@ -0,0 +1,649 @@
...
+            <td>Certified Ubuntu Devices</td>
+            <td class="u-align--center">$225</td>
+            <td class="u-align--center">$750</td>
+            <td class="u-align--center">$1,500</td>
+          </tr>
+
+          <tr style="background-color: #e5e5e5">
+            <td>Phone and ticket support</td>
+            <td class="u-align--center">-</td>
+            <td class="u-align--center">Monday - Friday</td>
```
Comment (pmahnke) [inline templates/pricing/devices.html]: this should also be 24/5

File: templates/pricing/devices.html (hunk 1 above)
Comment (sowasred2012) [reply templates/pricing/devices.html]: Sorry, missed the JS include - I've included the "general" contact form partial, there isn't one specific to devices (whereas there is for infra, for example), should we have one created?

Comment (pmahnke) [reply templates/pricing/devices.html]: general is ok for now, but we should get one for this page particularly

**Comment by sowasred2012** [comment]
@anasereijo could you take another look at this when you have a sec?

**Comment by anasereijo** [comment]
Hi @sowasred2012 

Desktop looks good!

Mobile:

- [ ] hide ubuntu certified image

- [ ] 'Security updates' table doesn't look very good on mobile. Anyway you can fix it?

**Comment by anasereijo** [comment]
@sowasred2012 looks good 👍

**Pull request merged**